src/node/cli.ts

```diff
@@ -510,13 +509,20 @@ export async function readConfigFile(configPath?: string): Promise<ConfigArgs> {
     }
   }
 
-  if (!(await fs.pathExists(configPath))) {
-    await fs.outputFile(configPath, await defaultConfigFile())
+  try {
+    await fs.writeFile(configPath, await defaultConfigFile(), {
+      flag: "wx", // wx means to fail if the path exists.
+    })
     logger.info(`Wrote default config file to ${humanPath(configPath)}`)
+  } catch (error) {
+    // EEXIST is fine; we don't want to overwrite existing configurations.
+    if (error.code !== "EEXIST") {
+      throw error
+    }
   }
 
-  const configFile = await fs.readFile(configPath)
-  return parseConfigFile(configFile.toString(), configPath)
+  const configFile = await fs.readFile(configPath, "utf8")
```

Comment:
nice fix :+1: 

is there a way to configure linters to enforce usage of utf8?

Reply:
The linter does recognize this, I just personally prefer to pass "utf8" rather than call `toString()` afterward haha. I'm not sure if there's a performance benefit or anything though.

Reply:
Oh wait I might have misread. Like enforce using "utf8" instead of `toString()`? I'd love that.

Reply:
makes sense, I think configuring utf8 on the stream makes things easier to use... sometimes these encoding things can be non-obvious in code reviews and lead to bugs that only happen sometimes (i.e. not with English/Latin-script text), and unfortunately older languages like JavaScript and Java have bad defaults, since they began in a pre-Unicode era